.gitignore

```diff
@@ -47,3 +47,9 @@ win32/common
 **/runner.log
 **/runner.trs
 **/test-suite.log
+/x64
```

Comment:
Please avoid building directly in your source tree if you can.  It's important to keep the out-of-tree build working, which is an easy feature to break by accident unless constantly exercised.

Also, building out-of-tree is just cleaner.  It means you can always just discard your build tree without losing any work.  _And_ it means that you don't need to insert any files generated by your particular build environment into the `.gitignore`.